indexer/src/main/java/org/pdxfinder/dataloaders/UniversalLoaderOmic.java

```diff
@@ -332,7 +332,7 @@ else if(toBeCreatedMolcharNodes.containsKey(molcharKey)){
 
             boolean foundSpecimen = false;
 
-            if(sampleOrigin.toLowerCase().equals("patient tumor")){
+            if(sampleOrigin.toLowerCase().equals("patient tumor") || sampleOrigin.toLowerCase().equals("patient")){
```

Comment:
Sonar recommends using "equalsIgnoreCase()" to be more concise here.

Reply:
Yepp, fixed it!